src/System.Net.Http/src/System/Net/Http/Headers/HeaderUtilities.cs

```diff
@@ -388,7 +388,8 @@ internal static void DumpHeaders(StringBuilder sb, params HttpHeaders[] headers)
             //    HeaderName2: Value1
             //    ...
             // }
-            sb.Append("{\r\n");
+            sb.Append("{");
+            sb.Append(Environment.NewLine);
```

Comment:
This could also be:
```C#
sb.Append('{').Append(Environment.NewLine);
```
Same in other such cases below.

Reply:
Or `sb.AppendLine`

Reply:
Even better

Reply:
updated with `sb.AppendLine()`